test/functional/actions/resetEventMergeId.spec.js

```diff
@@ -10,14 +10,10 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import createExtensionViewController from "../helpers/createExtensionViewController";
+import extensionViewController from "../helpers/extensionViewController";
 import spectrum from "../helpers/spectrum2";
 import createFixture from "../helpers/createFixture";
 
-const extensionViewController = createExtensionViewController(
-  "actions/resetEventMergeId.html"
```

Comment:
The path here wasn't actually being used since I re-worked how tests are run. This means `createExtensionViewController` could just change to `extensionViewController`.